src/components/AddressSearch.js

```diff
@@ -127,6 +141,13 @@ const AddressSearch = (props) => {
                 description: [styles.googleSearchText],
                 separator: [styles.googleSearchSeparator],
             }}
+            onLayout={(event) => {
+                // We use the height of the element to determine if we should hide the border of the listView dropdown
+                // to prevent a lingering border when there are no address suggestions.
+                // The height of the empty element is 2px (1px height for each top and bottom borders)
+                const {height} = event.nativeEvent.layout;
```

Comment:
Style guidelines: avoid destructuring when it is only one variable you are pulling (https://github.com/Expensify/App/blob/main/STYLE.md#destructuring)

Reply:
Removed!